content/en/apps/tutorials/targets.md

```diff
@@ -16,53 +16,210 @@ This tutorial will take you through how to build target widgets.
     subtitle_translation_key: 'targets.all_time.subtitle',
-
     appliesTo: 'reports',
     appliesToType: ['assessment'],
     date: 'now'
   }
-];
 ```
 
-### 2. Define a Monthly Target Widget
+### 2. Define the Total Assessments Monthly Target Widget
+This widget counts the total number of assessment reports that have been submitted by the user for this month.
+Edit the `targets.js` file and add another target widget definition object to define the total assessments monthly widget as shown below:
 
-Edit the `targets.js` file and add another target widget definition object to define a monthly widget as shown below:
+```javascript
+  {
+    id: 'assessments-this-month',
+    type: 'count',
+    icon: 'icon-healthcare-assessment',
+    goal: 10,
+    translation_key: 'targets.assessments.title',
+    subtitle_translation_key: 'targets.this_month.subtitle',
+    appliesTo: 'reports',
+    appliesToType: ['assessment'],
+    date: 'reported'
+  }
+```
+
+{{% alert title="Note" %}} All-time widgets have the `date` property set to `now` while monthly widgets have the `date` property set to `reported`. {{% /alert %}}
+
+### 3. Define the Cough Count Widget
+This widget calculates the total number of patients assessed that have been indicated to have a cough this month, regardless of the number of reports submitted for them. Note that `idType` counts the contact IDs.
+Edit the `targets.js` file and add the target widget as shown below:
 
 ```javascript
+  {
+    id: 'total-contacts-with-cough-this-month',
+    type: 'count',
+    icon: 'icon-healthcare-assessment',
+    goal: -1,
+    translation_key: 'targets.assessments.total.cough.title',
+    subtitle_translation_key: 'targets.this_month.subtitle',
+    appliesTo: 'reports',
+    appliesToType: ['assessment'],
+    appliesIf: function (contact, report) {
+      return Utils.getField(report, 'group_assessment.cough') === 'yes';
+    },
+    idType: 'contact',
+    date: 'reported'
+  }
+```
+
+### 4. Define the Cough Percentage Widget
+This widget calculates the percentage patients assessed that have been indicated to have a cough this month, regardless of the number of reports submitted for them.
+Edit the `targets.js` file and add the target widget as shown below:
+
+```javascript
+  {
+    id: 'percentage-contacts-with-cough-this-month',
+    type: 'percent',
+    icon: 'icon-healthcare-assessment',
+    goal: -1,
+    translation_key: 'targets.assessments.percentage.cough.title',
+    percentage_count_translation_key: 'targets.assessments.percentage.with.cough',
+    subtitle_translation_key: 'targets.this_month.subtitle',
+    appliesTo: 'reports',
+    appliesToType: ['assessment'],
+    appliesIf: function (contact) {
+      return contact.contact && contact.contact.parent && contact.contact.parent.parent && contact.contact.parent.parent.parent;
+    },
+    passesIf: function(contact, report) {
+      return Utils.getField(report, 'group_assessment.cough') === 'yes';
```

Comment:
It would be useful to create a very simple form that shows the structure that would allow for the targets logic to work.

Showing a screenshots of the survey and choices tabs would be more than sufficient

Reply:
This could be referenced from the forms tutorials.

Reply:
I had referenced them [here as requirements](https://github.com/medic/cht-docs/pull/574/files#diff-bd3524c0eb48b41d008a94d557e540fe339bab6d98c5d85fbf0671b251355231R36).